src/Template.php

```diff
@@ -162,14 +162,29 @@ public function remove( $var_name ) {
 		return $this;
 	}
 
+	/**
+	 * Checks if template file exists
+	 *
+	 * @since  [next]
+	 * @return bool
+	 */
+	public function exists() {
+		return is_file( $this->get_path() );
```

Comment:
```suggestion
		return $this->fs->is_file( $this->get_rel_path() );
```

Let's use the internal filesystem instead